pkg/clients/generated/apis/alloydb/v1beta1/alloydbcluster_types.go

```diff
@@ -148,8 +148,8 @@ type ClusterRestoreContinuousBackupSource struct {
 
 type ClusterSecondaryConfig struct {
 	/* Name of the primary cluster must be in the format
-	'projects/{project}/locations/{location}/clusters/{cluster_id}'. */
-	PrimaryClusterName string `json:"primaryClusterName"`
```

Comment:
@justinsb Has the "old" field gone out in a release yet? Wondering if we need to retain "PrimaryClusterName" for backward compatability...

Reply:
It has not been part of any release, had to create another separate PR https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1377 for the thirdparty tfgpb changes

The PR https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1377 which adds this field was only submitted to main branch an hour ago